src/app/test-record/vehicle/vehicle-edit/vehicle-edit.component.html

```diff
@@ -0,0 +1,184 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-three-quarters">
+    <div class="govuk-form-group">
+      <fieldset class="govuk-fieldset">
+        <div class="govuk-form-group">
+          <div class="govuk-grid-row govuk-!-margin-bottom-3" [class.govuk-form-group--error]="formErrors?.includes('Enter the vin')">
+            <div class="govuk-grid-column-one-half">
+              <label class="govuk-label govuk-!-font-weight-bold">
+                Vehicle identification number (VIN)
+              </label>
+              <span *ngIf="formErrors?.includes('Enter the vin')" class="govuk-error-message">
```

Comment:
<span id="your-error-id"
                          class="govuk-error-message" *ngIf="testResultChildForm.get('vin').invalid">
                <span class="govuk-visually-hidden">Error:</span>
                <span>Enter the VIN</span>
              </span>